libs/neo4j/tests/unit_tests/llms/fake_llm.py

```diff
@@ -59,3 +59,6 @@ def _get_next_response_in_sequence(self) -> str:
         response = queries[list(queries.keys())[self.response_index]]
         self.response_index = self.response_index + 1
         return response
+
+    def bind_tools(self, tools: Any) -> None:
```

Comment:
How is this method used?

Reply:
In a real LLM, it's used to provide the LLM with [tools](https://python.langchain.com/docs/integrations/tools/) to use. Some LLMs don't support tool use, however, so the `GraphCypherQAChain` checks if the LLM has the `bind_tools` method when you initialise it with the `use_function_response` parameter and raises an error if not. I've just defined a dummy method here for this fake LLM so it can pass that check and `test_graph_cypher_qa_chain_prompt_selection_6` can run.